powerdnsadmin/models/user.py

```diff
@@ -317,26 +303,21 @@ def is_validate(self, method, src_ip='', trust_user=False):
                                         .format(self.username))
                                     return False
 
-                                for group in [
-                                        g.decode("utf-8")
-                                        for g in user_ad_member_of
-                                ]:
-                                    user_ldap_groups += self.ad_recursive_groups(
-                                        group)
+                                user_ad_member_of = [g.decode("utf-8") for q in user_ad_member_of]
```

Comment:
There is a typo here and we are good to go. Can you squash the two changes commit with the next added to fix this line in one and force push so we have two commits to merge ?

I think we'll be good.

Reply:
I believe it should be ok now. Please let me know If you find any other problems.

Reply:
I'll check it soon, it looks ok for me thank you.